src/WikiService.Models.Dto/Requests/Article/EditArticleRequest.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace LT.DigitalOffice.WikiService.Models.Dto.Requests.Article
+{
+  public record EditArticleRequest
```

Comment:
There is no need in record if u don't use its features :)
Whole this text can be simplified to:

`
using System;

namespace LT.DigitalOffice.WikiService.Models.Dto.Requests.Article;

public record EditArticleRequest(string Name, string Content, bool IsActive, Guid RubricId);
`

Reply:
OK, fixed. thanks)